3drepo/3drepo.io
# Issue 4712 - ticket groups bugs

This fixes #4712

#### Description
- The highlighted objects can be hidden. If that's the case, the equivalent group is not created on save/update current view
- Groups via save/update current view in a new ticket were missing the key

#### Test cases
Click on "new ticket" and select a template with view and:
1. Highlight a mesh
2. Hide that mesh
3. Save the current view
> No error should be triggered
4. Open the groups (there should be a hidden group)
5. Edit the hidden group
> The group should be edited correctly and no other groups should be created
